src/views/Team.vue

```diff
@@ -1,9 +1,31 @@
 <template>
    
     <div class=body>
-        <h1 class=heading>Meet the Doers and Builders!</h1>
+        <h1 class=heading style="font-size:55px;font-family:san-serif;margin-top:0px;">Meet the Doers and Builders!</h1>
         <br>
-        <div class=container>
+        
+        <div class=container1>
+            <div class=our_team style="width:230px;">
+                <div class=pic>
+                    <img src="../assets/our+team/4.jpg" alt="error" style="width:150px;height:150px;">
+                </div>
+                <div class=team-content>
+                    <h4 class=title>Surajit Kumar Roy</h4>
+                    <span class="post">Project Instructor</span>
+                </div>
+                <ul class="social">
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fab fa-facebook"></i></a></li>
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fab fa-instagram"></i></a></li>
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fas fa-envelope"></i></a></li>
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fab fa-linkedin"></i></a></li>
```

Comment:
4 "Facebook page"? forgot to edit, perhaps

Reply:
Yeah, I forgot to edit!